domain/consensus/processes/transactionvalidator/test_transaction_validator.go

```diff
@@ -0,0 +1,22 @@
+package transactionvalidator
+
+import (
+	"github.com/kaspanet/kaspad/domain/consensus/model"
+	"github.com/kaspanet/kaspad/domain/consensus/utils/txscript"
+)
+
+type testTransactionValidator struct {
+	*transactionValidator
+}
+
+func NewTestTransactionValidator(baseTransactionValidator model.TransactionValidator) *testTransactionValidator {
+	return &testTransactionValidator{transactionValidator: baseTransactionValidator.(*transactionValidator)}
+}
+
+func (tbv *testTransactionValidator) SigCache() *txscript.SigCache {
+	return tbv.sigCache
+}
+
+func (tbv testTransactionValidator) SetSigCache(sigCache *txscript.SigCache) {
```

Comment:
Please be consistent in receiver types (as in, this receiver should be `*testTransactionValidator`)